meta-facebook/yv4-sd/src/platform/plat_pdr_table.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef PLAT_PDR_TABLE_H
+#define PLAT_PDR_TABLE_H
+
+#include <stdint.h>
+
+#define MAX_SENSOR_SIZE 60
+
+uint8_t plat_get_pdr_size();
+void load_pdr_table(void);
```

Comment:
Better to use consistent plat_ prefix (if possible, i.e. doesn't involve much change in other places).